apps/discord-bot-internal/src/types/twitch.ts

```diff
@@ -0,0 +1,33 @@
+export interface TwitchStream {
```

Comment:
We have a smaller version of this type in `models.ts` already that the frontend uses, I would move both of these there.